tools/podmanlinkchecker.sh

```diff
@@ -1,8 +1,22 @@
 #!/usr/bin/env sh
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
 set -ex
 
 echo "Run linkchecker on a running che-docs container"
 
-podman exec -ti \
-  che-docs \
-  "./tools/linkchecker.sh" \
+if command -v podman
+  then RUNNER=podman
+elif command -v docker
+  then RUNNER=docker
+else echo "No installation of podman or docker found in the PATH" ; exit 1
+fi
+
+${RUNNER} exec -ti che-docs "./tools/linkchecker.sh"
```

Comment:
Hello, why not use the official image name there ? `quay.io/eclipse/che-docs:latest` ?
so by default people may not have to build image before

Reply:
It's reusing the image launched by podmanpreview (launched with name che-docs) as it is a prerequisite to have the the site built and running.

Reply:
(So there is no image build involved)

Reply:
OK understood about exec vs run but why not having a single command ? like not having to have container running all the time.